packages/firestore/src/core/firestore_client.ts

```diff
@@ -522,6 +522,21 @@ export class FirestoreClient {
     });
   }
 
+  /**
+   * Returns a Promise that resolves when when all writes that were pending at the time when this
```

Comment:
s/when when/when/
s/at the time when/at the time/

Reply:
Done.